build.gradle

```diff
@@ -1,106 +1,23 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
 buildscript {
   repositories {
-    mavenCentral()
-    mavenLocal()
-    maven {
-      url 'https://plugins.gradle.org/m2/'
-    }
+    jcenter()
   }
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.2.3'
-    classpath 'com.github.dcendents:android-maven-plugin:1.2'
-    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
-    classpath 'net.researchgate:gradle-release:2.4.0'
-  }
-}
-
-apply plugin: 'com.android.library'
-apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.neenbedankt.android-apt'
-apply plugin: 'checkstyle'
-apply plugin: 'net.researchgate.release'
-
-group = GROUP
-version = VERSION_NAME
-
-release {
-  versionProperties = ['VERSION_NAME']
-}
-
-afterReleaseBuild.dependsOn uploadArchives
-
-defaultTasks = ['clean', 'assembleDebug', 'checkstyle', 'copyTask', 'testDebug', 'install']
```

Comment:
After re-adding default tasks locally I ran into the following compile error in the tests

```
/usr/local/code/pelias-android-sdk/lib/src/test/java/com/mapzen/pelias/widget/PeliasSearchViewTest.java:58: error: cannot access AndroidHttpClient
    shadowOf(queryText).setViewFocus(true);
    ^
  class file for android.net.http.AndroidHttpClient not found
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error
4 warnings
:lib:compileDebugUnitTestJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':lib:compileDebugUnitTestJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```